Adding recommended tools doc (#26)
esdc-devcop/strategy: recommendation/tools.md

# Recommended Tools

> This is a list of Tools that are recommended for use in ESDC. 

Reviewer: ```suggestion
This is a list of tools that are recommended for use in ESDC. 
```

> The topic identifies the area of work that the tool is recommened for, 

Reviewer: ```suggestion
The topic identifies the area of work that the tool is recommended for, 
```

> and the scope may identify a particular streem of development that that tool is recommened for.

Reviewer: ```suggestion
and the scope may identify a particular stream of development that that tool is recommended for.
```

<table>
  <thead>
    <tr>
      <th>Topic</th>
      <th>Scope</th>
      <th>Tool</th>
      <th>References</th>
    </tr>
  </thead>
  <tbody>
    <tr>

>       <td rowspan=2>Source control</td>

Reviewer: ```suggestion
      <td rowspan=2>Source Control</td>
```

        <a href="https://github.com/gcdevops/strategie-devops-strategy/blob/WorkInProgress/Content/OfficalGCCodeSupport.md">https://github.com/gcdevops/strategie-devops-strategy/blob/WorkInProgress/Content/OfficalGCCodeSupport.md</a><br />
        <a href="https://esdc-devcop.github.io/strategy/guides/tfs-to-gccode.html" rel="nofollow">https://esdc-devcop.github.io/strategy/guides/tfs-to-gccode.html</a></td>
    </tr>
    <tr>
      <td>Logging</td>
      <td>.NET</td>
      <td>NLog</td>
      <td><a href="http://nlog-project.org/" rel="nofollow">http://nlog-project.org/</a></td>
    </tr>
    <tr>
      <td>Asset Management</td>
      <td>-</td>
      <td>Artifactory</td>
      <td><a href="https://jade-repos.intra.dev/artifactory/webapp/#/home" rel="nofollow">https://jade-repos.intra.dev/artifactory/webapp/#/home</a><br/>
        <a href="https://discourse.devx.team/t/artifactory-as-an-asset-management-and-vulnerability-management-process/89" rel="nofollow">https://discourse.devx.team/t/artifactory-as-an-asset-management-and-vulnerability-management-process/89</a></td>
    </tr>
    <tr>
      <td>Programming Language</td>
      <td>-</td>

>       <td>typeScript</td>

Reviewer: ```suggestion
      <td>TypeScript</td>
```
